### PR TITLE
Loader: Add colorspace entry to Version Info

### PR DESCRIPTION
**Pull request by fabiaserra**
## Changelog Description
Small addition to add "Colorspace" entry in Version Info tab in loader if version has colorspace data

image

## Testing notes:
1. Open the loader and click on a Version entry that has colorspace data

### CODE REVIEW

**Comment by mustafa-zarkash** [comment]
I've tested it in Houdini and Maya, and this PR doesn't seem to affect them.

File: openpype/tools/loader/widgets.py [hunk 1]
```diff
@@ -741,6 +741,16 @@ def set_version(self, version_doc=None, version_id=None):
             "source": source_label
         }
 
+        # Add colorspace info to Version Info tab
+        version_colorspace = version_doc["data"].get("colorspace")
+        colorspace_html = ""
```
Comment (BigRoy) [inline openpype/tools/loader/widgets.py:746]: Isn't the colorspace on the representations instead of the version?

**Comment by BigRoy** [comment]
> I've tested it in Houdini and Maya, and this PR doesn't seem to affect them.

It should affect them, but only for publishes that have colorspace data written into them. If you're not seeing the information right hand side even if the colorspace data exists on the version/representations than I'd say it's a bug.

File: openpype/tools/loader/widgets.py (hunk 1 above)
Comment (fabiaserra) [reply openpype/tools/loader/widgets.py:746]: I checked a few entries in MongoDB and I can only see it on the version data

**Comment by fabiaserra** [comment]
> > I've tested it in Houdini and Maya, and this PR doesn't seem to affect them.
> 
> It should affect them, but only for publishes that have colorspace data written into them. If you're not seeing the information right hand side even if the colorspace data exists on the version/representations than I'd say it's a bug.

Exactly

File: openpype/tools/loader/widgets.py (hunk 1 above)
Comment (jakubjezek001) [reply openpype/tools/loader/widgets.py:746]: In fact @BigRoy is right @fabiaserra. The new colorspaceData distribution is kept on representation level. The version data colorspace key is still available but will be removed soon.

**Comment by jakubjezek001** [comment]
I am not sure how to put my mind into this. I appreciate your attempt @fabiaserra but I am afraid this will only work for couple months only. I am suggesting to close it. But nothing will be broken if we merge it ;)

File: openpype/tools/loader/widgets.py (hunk 1 above)
Comment (fabiaserra) [reply openpype/tools/loader/widgets.py:746]: yeah fair enough, no big deal, it was just a fast utility for me while testing the OpenRV integration to make sure what I was loading was getting the expected colorspace, happy to close it

**Pull request closed without merging**